cmake/clang-tidy.cmake

```diff
@@ -0,0 +1,36 @@
+option(ENABLE_CLANG_TIDY "Add run-clang-tidy automatically to builds" OFF)
+find_program(CLANG_TIDY_EXE
+        NAMES run-clang-tidy.py run-clang-tidy run-clang-tidy-7
+        HINTS ${CMAKE_CURRENT_SOURCE_DIR}/scripts
+        DOC "Path to clang-tidy executable")
+if(CLANG_TIDY_EXE)
```

Comment:
Probably should be renamed to TIDY_RUNNER instead OF CLANG_TIDY_EXE

Reply:
OK